swap/swap.go

```diff
@@ -419,6 +421,103 @@ func (s *Swap) Balances() (map[enode.ID]int64, error) {
 	return balances, nil
 }
 
+// Cheques returns all known last sent and received cheques, grouped by peer
+func (s *Swap) Cheques() (map[enode.ID]map[string]*Cheque, error) {
```

Comment:
Right now, all the functions from the `swapAPI` are defined in the file swap.go. Don't you think it would make more sense to define all these in a separate file, such that both swap.go and swap_test.go are a bit leaner? swap.go, then can be only for the swap internal logic.

Reply:
it's definitely arguable, but it should be a separate PR if we go ahead with it.

Reply:
@Eknir please create an issue for this if you think it's necessary and link it here. thanks

Reply:
Fully agree here. Let's keep `swap.go` for actual swap functionality.

Reply:
#1882